vscode-extension/README.md

```diff
@@ -36,22 +38,35 @@ The following elements of a schema file are highlighted:
 
 - Model Blocks (`model`)
 - Config Blocks (`datasource` and `generator`)
-- Enum Blocks (`enum`)
+- Enum Blocks (`enum`) (not supported yet)
```

Comment:
Please create an issue for this in the appropriate repo if it does not exist yet, if it does maybe link it from here.

Reply:
should I create a new PR for this, since the branch is already merged?

Reply:
I double checked: this is supported, just not for sqlite as sqlite does not support enums. I'll add a screenshot for that